AppCenterDataStorage/AppCenterDataStorage/MSDictionaryDocument.m

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+#import "MSDictionaryDocument.h"
+
+@implementation MSDictionaryDocument
+
+@synthesize dictionary = _dictionary;
```

Comment:
You don't need to synthesize when the property is declared in an interface ;).

Reply:
```suggestion
```